app/assets/stylesheets/language.scss

```diff
@@ -101,3 +103,14 @@
     background-color: #f5f5f5;
   }
 }
+
+@media (max-width: 400px) {
```

Comment:
these is no `$screen-xs-min` since that's the default. Can we add `.container-fluid` on it's parent `.row` and potentially avoid using this style all together?